vm/src/import.rs

```diff
@@ -46,39 +43,6 @@ pub fn import_builtin(vm: &VirtualMachine, module_name: &str) -> PyResult {
         })
 }
 
-pub fn import_module(vm: &VirtualMachine, current_path: PathBuf, module_name: &str) -> PyResult {
-    // Cached modules:
-    let sys_modules = vm.get_attribute(vm.sys_module.clone(), "modules").unwrap();
-
-    // First, see if we already loaded the module:
-    if let Ok(module) = sys_modules.get_item(module_name.to_string(), vm) {
-        Ok(module)
-    } else if vm.frozen.borrow().contains_key(module_name) {
-        import_frozen(vm, module_name)
-    } else if vm.stdlib_inits.borrow().contains_key(module_name) {
-        import_builtin(vm, module_name)
-    } else if cfg!(feature = "rustpython-compiler") {
```

Comment:
The point of this feature flag was to check if rustpython was built with bytecode compiler support. If not, the module cannot be loaded from disc. I think importlib.py has a similar mechanism. Please refer to the function `imp_exec_builtin` in `imp.rs`. Maybe the logic deleted here, can be replicated over there?

Reply:
Only the `external_importers` use compile now. So I changed `init_importlib` to not install them if there is not compiler.